### PR TITLE
Siva, Bindu | BAH-1378| Add latest bahmni 0.94 dependencies for log4j2 upgrade

### PR DESCRIPTION
**Pull request by binduak**
JIRA card

[https://bahmni.atlassian.net/browse/BAH-1378](https://bahmni.atlassian.net/browse/BAH-1378)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -66,6 +67,22 @@
...
+			</dependency>
+
+			<dependency>
+				<groupId>org.bahmni.module</groupId>
+				<artifactId>bahmni-extension</artifactId>
+				<version>${bahmniVersion}</version>
+				<type>jar</type>
+
+			</dependency>
+
```
Comment (angshu) [inline pom.xml:85]: openmrs-api and core versions to be upgraded to 2.1.5. change the "openMRSVersion" variable

Comment (binduak) [reply pom.xml:85]: Done

File: api/pom.xml [hunk 2]
```diff
@@ -18,7 +18,7 @@
         <dependency>
             <groupId>org.bahmni.module</groupId>
             <artifactId>bahmni-migrator</artifactId>
-            <version>0.92-SNAPSHOT</version>
+            <version>${bahmniVersion}</version>
```
Comment (angshu) [inline api/pom.xml:21]: I think the variable is better named as bahmniJavaUtils version, no?  this is about the libs from bahmni-java-utils, right?

File: pom.xml [hunk 3]
```diff
@@ -28,10 +28,12 @@
 	</modules>
 
 	<properties>
-		<openMRSVersion>2.0.3</openMRSVersion>
+		<openMRSVersion>2.1.6</openMRSVersion>
+		<bahmniVersion>0.94-SNAPSHOT</bahmniVersion>
```
Comment (angshu) [inline pom.xml:32]: Why not use 0.94.1 for java utils. See also comment on the vairable name above

File: api/pom.xml (hunk 2 above)
Comment (rohit-yawalkar) [reply api/pom.xml]: Yes @angshu, we changed variable name to bahmniJavaUtils, as it is pointing to libs from bahmni-java-utils

File: pom.xml (hunk 3 above)
Comment (rohit-yawalkar) [reply pom.xml]: @angshu, We modified version to 0.94.1

**Review by angshu — APPROVED** [review]

**Pull request merged**